ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/CatalogController.java

```diff
@@ -216,17 +218,26 @@ private Map<String, String> authConvertFromServerToMeta(
             AUTH_CONFIGS_KEY_PRINCIPAL, serverAuthConfig.get(AUTH_CONFIGS_KEY_PRINCIPAL));
         break;
       case AUTH_CONFIGS_VALUE_TYPE_AK_SK:
-        CatalogUtil.copyProperty(serverAuthConfig, metaAuthConfig, AUTH_CONFIGS_KEY_ACCESS_KEY);
-        CatalogUtil.copyProperty(serverAuthConfig, metaAuthConfig, AUTH_CONFIGS_KEY_SECRET_KEY);
+        CatalogUtil.copyProperty(
+            serverAuthConfig,
+            catalogMeta.getCatalogProperties(),
+            AUTH_CONFIGS_KEY_ACCESS_KEY,
+            S3FileIOProperties.ACCESS_KEY_ID);
+        CatalogUtil.copyProperty(
+            serverAuthConfig,
+            catalogMeta.getCatalogProperties(),
+            AUTH_CONFIGS_KEY_SECRET_KEY,
+            S3FileIOProperties.SECRET_ACCESS_KEY);
         break;
     }
-    return metaAuthConfig;
+    catalogMeta.setAuthConfigs(metaAuthConfig);
   }
 
   /** Convert meta auth config to server auth config DTO */
   private Map<String, Object> authConvertFromMetaToServer(
```

Comment:
`pareAuthConfigsFromCatalogMeta` may be a proper method name.

Reply:
I noticed that we should not remove the auth properties from the catalog properties of `CatalogMeta`, which will  destroy the object. So `extractAuthConfigsFromCatalogMeta` may be better.

Reply:
Sorry for my spelling mistake. I originally meant `parseAuthConfigsFromCatalogMeta`, but `extractAuthConfigsFromCatalogMeta` looks good to me too.

Reply:
Now, I remove the auth config from properties of `CatalogSettingInfo` and `CatalogRegisterInfo` during `removeHiddenProperties`.